src/pybind/mgr/dashboard/module.py

```diff
@@ -517,6 +517,12 @@ class Root(object):
             def default(self, *args, **kwargs):
                 if module.get_module_option('standby_behaviour', 'redirect') == 'redirect':
                     active_uri = module.get_active_uri()
+
+                    if cherrypy.request.path_info == '/api/prometheus_receiver':
+                        module.log.info("Suppressed redirecting alert to active '%s'",
```

Comment:
The audit log is not logging this? Do we need an extra log here? This is probably going to repeat quite often right?

Reply:
Yes, I agree. Should I possibly change it to `module.log.debug`?

Reply:
Ok.